lighthouse-core/test/network-records-to-devtools-log.js

```diff
@@ -74,9 +78,12 @@ function getRequestWillBeSentEvent(networkRecord, index) {
  */
 function getResponseReceivedEvent(networkRecord, index) {
   const headers = headersArrayToHeadersDict(networkRecord.responseHeaders);
-  const timing = networkRecord.timing;
-  if (timing && timing.requestTime === undefined) {
-    timing.requestTime = networkRecord.startTime || 0;
+  let timing;
+  if (networkRecord.timing) {
+    timing = {...networkRecord.timing};
+    if (timing.requestTime === undefined) {
+      timing.requestTime = networkRecord.startTime || 0;
+    }
```

Comment:
a few tweaks to get this to work well with existing tests. e.g. not overwriting `timing` or `initiator` as tests sometimes reuse the object and expect it to be unchanged